src/TorchSharp/TorchVision/IO/Image.cs

```diff
@@ -1,3 +1,4 @@
+// Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
 using System.IO;
```

Comment:
Many of datasets in `torchvision.datasets` return `PIL` image with `__getitem__()`. Which TorchSharp's datasets should return, `Tensor` or objects of `Imager`?

Reply:
Tensor.